Documentation/ContentObjects/Image/Index.rst

```diff
@@ -755,26 +786,25 @@ Responsive/adaptive rendering
    60.layoutKey = srcset
 
 
-This returns as an example all per default possible HTML-Output:
+This returns as an example all per default possible HTML output:
```

Comment:
The sentence sounds weird for me. But I have no other suggestion by now. But this is not a stopper for this PR.